web/src/api/typings/tasks.ts

```diff
@@ -1,6 +1,6 @@
 import { ValidationType } from 'api/typings';
 
-export type Task = {
+export type ApiTask = {
```

Comment:
Can you clarify for me please: does taskApi mean a task that comes from the backend?

Reply:
yes